packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -51,9 +52,12 @@ describe('KeyringController', () => {
     keyrings: Keyring[];
   };
   const additionalKeyrings = [QRKeyring];
+  const additionalKeyringBuilders = additionalKeyrings.map((keyringType) =>
+    keyringBuilderFactory(keyringType),
+  );
   const baseConfig: Partial<KeyringConfig> = {
     encryptor: new MockEncryptor(),
-    keyringTypes: additionalKeyrings,
+    keyringBuilders: additionalKeyringBuilders,
```

Comment:
Relevant changelog entry in `@metamask/eth-keyring-controller`: https://github.com/MetaMask/KeyringController/blame/main/CHANGELOG.md#L22